packages/SwingSet/src/kernel/vatAdmin/vatAdminWrapper.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * The VatAdmin wrapper vat.
+ *
+ * This is the only vat that has a direct pointer to the vatAdmin device, so it
+ * must act as a membrane, ensuring that only data goes in and out. It's also
+ * responsible for turning device affordances into objects that can be used by
+ * code in other vats.
+ */
+
+import harden from '@agoric/harden';
+
+export default function setup(syscall, state, helpers) {
+  function makePromiseForVat() {
+    let res;
+    let reject;
+    const p = new Promise((rsv, rj) => {
+      res = rsv;
+      reject = rj;
+    });
+    return harden({ p, res, reject });
+  }
+
+  function build(E, D) {
+    const vatIdsToRoots = new Map();
+
+    function createVatAdminService(vatAdminNode) {
+      return harden({
+        createVat(code) {
+          const vatId = D(vatAdminNode).create(code);
+          if (!vatId.startsWith('v')) {
```

Comment:
I don't understand the v- vs non-v- check

Reply:
This is the error message. I added a comment to make it clearer.

Reply:
resolved (to fix later in #445)